packages/web-components/src/components/rux-slider/test/index.html

```diff
@@ -104,7 +104,7 @@ <h2>Prop/Slot Testing</h2>
             <script>
                 const log = document.getElementById('log')
                 const ticks = document.getElementById('ticks')
-                ticks.axisLabels = ['0', '25', '50', '75', '100']
+                ticks.axisLabels = ['Low', 'Med', 'High', 'Oh Noes!']
```

Comment:
whats going on here?

Reply:
@markacianfrani @FMorrison87 OH NO! I'm sorry. 🤣 Jonathan and I were having a side discussion about how ticks work and get labeled in slider and I wrote up something and shared screen but didn't think I committed it! How embarrassing. Let me fix.